scripts/maps.js

```diff
@@ -198,23 +198,35 @@ function createVenueList(venues) {
     var venueListEle = document.getElementById('venue-list');
 
     venues.forEach(function(venue) {
-        var node = document.createElement("li"),
+
+        console.log('venue', venue)
```

Comment:
This is fine for now, but just tidy up any unneeded `console.log`s as you go, otherwise the console is awash with remnants of old debugging.

Reply:
Oh yes! Will do